application/helpers/userstatistics_helper.php

```diff
@@ -2621,7 +2621,7 @@ public function generate_statistics($surveyid, $allfields, $q2show = 'all', $use
             }
 
             // Creating the first worksheet
-            $this->sheet = $this->workbook->addWorksheet(utf8_decode('results-survey' . $surveyid));
+            $this->sheet = $this->workbook->addWorksheet(mb_convert_encoding('results-survey' . $surveyid, 'ISO-8859-1'));
```

Comment:
https://www.php.net/manual/en/function.mb-convert-encoding.php 

Then must be `mb_convert_encoding('results-survey' . $surveyid, 'ISO-8859-1', 'UTF-8')` no ?